samples/TestApplicationShared/App.cs

```diff
@@ -18,35 +18,35 @@ public class App : Application
     {
         public App()
         {
-			// TODO: I believe this has to happen before we select sub systems. Can we
-			// move this safely into Application itself? Is there anything in here
-			// that is platform specific??
-			//
+            // TODO: I believe this has to happen before we select sub systems. Can we
+            // move this safely into Application itself? Is there anything in here
+            // that is platform specific??
+            //
```

Comment:
Ok. Is there any reason why it needs to remain virtual? It seems like we want core/foundation services registered always. Custom/switchable stuff is done via the application extensions or other DI?